OxfordBodleian/Juel-Jensen/BDLaethb6.xml

```diff
@@ -40,9 +43,8619 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            >Commemoration of Bālān</title>
+                                    </msItem>
+                                </msItem>
+
+                                <msItem xml:id="ms_i2.2.23">
+                                    <locus from="25rb" to="25va"/>
+                                    <title type="complete" ref="LIT4035SenkessarDL#Mask23"/>
+
+                                    <msItem xml:id="ms_i2.2.23.1">
+                                        <locus from="25rb" to="25va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Mask23Awnabyos">Commemoration of
+                                            ʾAwnābyos and ʾƎndrǝyās</title>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i2.2.23.2">
+                                        <locus target="#25va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Mask23SalAwnabya">Salām-hymn to
+                                            ʾAwnābyos and ʾƎndrǝyās</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i2.2.23.3">
+                                        <locus target="#25va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Mask23Ewostatewos">Commemoration
+                                            of St Eustace</title>
+                                    </msItem>
+                                 
+                                    <msItem xml:id="ms_i2.2.23.4">
+                                        <locus target="#25va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Mask23SalEwostatewos">Salām-hymn
+                                            to St Eustace</title>
+                                    </msItem>
+                                </msItem>
+
+                                <msItem xml:id="ms_i2.2.24">
+                                    <locus from="25va" to="26rb"/>
+                                    <title type="incomplete" ref="LIT4035SenkessarDL#Mask24"/>
+
+                                    <msItem xml:id="ms_i2.2.24.1">
+                                        <locus from="25va" to="26ra"/>
+                                        <title type="incomplete"
+                                            ref="LIT4035SenkessarDL#Mask24Gorgoryos">Commemoration
+                                            of Gorgoryos</title>
+                                        <note>The part of the text covered by the later added miniature is missing.</note>
```

Comment:
Something in the wording sounds unnatural to me. @gnizla what do you think?
Perhaps "Part of the text is missing because covered by a miniature added at a later time" ?